internal/config/duration.go

```diff
@@ -13,7 +13,7 @@ const (
 	Hour        Duration = Duration(time.Hour)
 )
 
-func (d *Duration) UnmarshalText(text []byte) error {
```

Comment:
These functions seem to not be used at all. Can we delete `UnmarshalText` and `NewTicker`? @aDotInTheVoid